### PR TITLE
Refactor Jenkinsfiles

### PR DESCRIPTION
**Pull request by eidenyoshida**

### CODE REVIEW

File: docker/dockerfile-build-centos [hunk 1]
```diff
@@ -2,14 +2,15 @@
 ARG base_image
 
 FROM ${base_image}
-LABEL maintainer="Nico Trost"
+LABEL maintainer="rochpcg-maintainer@amd.com"
```
Comment (eidenyoshida) [inline docker/dockerfile-build-centos:5]: @saadrahim is this the correct maintainer

Comment (saadrahim) [reply docker/dockerfile-build-centos:5]: yes, but you will need to make a distribution list for it, it doesn't exist now.

File: .jenkins/precheckin.groovy [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import com.amd.docker.*
+import java.nio.file.Path;
+
+def runCI = 
+{
+    nodeDetails, jobName->
+
+    def prj = new rocProject('rocHPCG', 'PreCheckin')
+    prj.paths.build_command = './install.sh -t --with-openmp=OFF --with-mpi=OFF'
+    prj.libraryDependencies = ['rocPRIM']
```
Comment (saadrahim) [inline .jenkins/precheckin.groovy:13]: What does this line do?

Comment (eidenyoshida) [reply .jenkins/precheckin.groovy:13]: I actually don't think it's needed. I will remove it

**Pull request merged**